lib/runtime-core/src/typed_func.rs

```diff
@@ -258,11 +258,6 @@ where
             _phantom: PhantomData,
         }
     }
-
-    /// Get the underlying func pointer.
-    pub fn get_vm_func(&self) -> NonNull<vm::Func> {
-        self.func
-    }
```

Comment:
Is this a breaking change?

Reply:
Nop. It was implemented for `Wasm` only, now it's implemented for `Wasm` and `Host`.